node/src/command.rs

```diff
@@ -48,13 +48,12 @@ impl SubstrateCli for Cli {
 
 	fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
 		Ok(match id {
-			"dev" => Box::new(chain_spec::development_config()?),
-			"" | "local" => Box::new(chain_spec::local_testnet_config()?),
-			path => Box::new(chain_spec::ChainSpec::from_json_file(
-				std::path::PathBuf::from(path),
-			)?),
+		  "" | "local_testnet" => Box::new(chain_spec::local_testnet::local_testnet()?),
+		  "dev" => Box::new(chain_spec::dev::dev()?),
+		  "test_valiu_testnet" => Box::new(chain_spec::test_valiu_testnet::test_valiu_testnet()?),
```

Comment:
I guess it can already be jus "testnet" and now that we are it what do you say about the function on each of the modules called spec so we do `chain_spec::testnet::spec()` ... just a tiny OCD on my side 😆

Reply:
Changed